src/components/CityButton.js

```diff
@@ -3,11 +3,9 @@ import { Text, View, StyleSheet, TouchableOpacity } from 'react-native'
 import { MaterialCommunityIcons } from '@expo/vector-icons'; 
 
 
-const CityButton = ({toOpenModal}) => {
+const CityButton = ({onPress}) => {
   return (
-    <TouchableOpacity 
-      onPress={toOpenModal}
-    >
+    <TouchableOpacity onPress={onPress}>
       <View style={styles.container}>
```

Comment:
Change the names of simple callback functions to 'onPress', to make things more simple.